src/bin/CMakeLists.txt

```diff
@@ -250,6 +250,9 @@ elseif (CX_PLATFORM STREQUAL "darwin")
         PROPERTIES LINK_FLAGS "-exported_symbols_list \"${CMAKE_CURRENT_SOURCE_DIR}/darwin/exports.txt\"")
 endif()
 
+include(GNUInstallDirs)
+set(include_dest ${CMAKE_INSTALL_INCLUDEDIR})
```

Comment:
So, what exactly does this do? Can you update the PR description a bit more to explain too? Thanks!